src/main/kotlin/org/rust/ide/presentation/Utils.kt

```diff
@@ -60,13 +61,40 @@ fun getPresentationForStructure(psi: RsElement): ItemPresentation {
     return PresentationData(presentation, null, icon, null)
 }
 
+private fun buildTemplateImplTypeName(psi: RsImplItem): String? {
+    val paramsList = ArrayList<String>()
+    val baseName = psi.typeReference?.baseType?.name ?: return null
+    for(typeParameter in psi.typeParameters) {
+        val name = typeParameter.name ?: continue
+        val typeBounds = typeParameter.typeParamBounds?.childrenOfType<RsPolybound>()
+        if(typeBounds != null) {
+            for(bound in typeBounds) {
+                val value = bound.bound.traitRef?.path?.referenceName
+                if(value != null) {
+                    val builder = StringBuilder()
+                    builder.append(name)
+                    builder.append(": ")
+                    builder.append(value)
+                    paramsList.add(builder.toString())
+                    break
```

Comment:
Is it intentional that the code here does not handle multiple type bounds?
This code
```rust
impl<K: Display + Ord> A<K> {
    pub fn ccc() {}
}
```
shows up the same way as
```rust
impl<K: Display> A<K> {
    pub fn aaa() {}
}
```